packaging/test_plugin_unified_storage_tiering.py

```diff
@@ -213,6 +213,15 @@ def invoke_storage_tiering_rule():
     with session.make_session_for_existing_admin() as admin_session:
         admin_session.assert_icommand(['irule', '-r', rep_instance, '-F', rule_file_path])
 
+def get_tracked_replica(session, logical_path):
+    coll_name = os.path.dirname(logical_path)
+    data_name = os.path.basename(logical_path)
+
+    tracked_replica_query = 'select META_DATA_ATTR_UNITS where DATA_NAME = \'{}\' and COLL_NAME = \'{}\' ' \
+        'and META_DATA_ATTR_NAME = \'irods::storage_tiering::group\''.format(data_name, coll_name)
```

Comment:
this could be another string in the wild...
https://github.com/irods/irods_capability_storage_tiering?tab=readme-ov-file#customizing-metadata-attributes

of course, not during the test... but wanted to flag this for your brain.

Reply:
Indeed. Might need to make it a parameter for any future tests that customize the terms.

Reply:
Done